tabular/tests/unittests/test_tabular.py

```diff
@@ -219,6 +220,7 @@ def test_advanced_functionality():
 
     assert(predictor.get_model_full_dict() == dict())
     predictor.refit_full()
+    predictor.plot_ensemble_model()
```

Comment:
can check that the output file exists on disk, probably that is good enough for testing purposes